code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -308,6 +308,20 @@
 	item_state = "utility"
 	use_item_overlays = 1 // So it will still show tools in it in case sec get lazy and just glance at it.
 
+/obj/item/storage/belt/military/traitor/hacker
+	name = "tool-belt"
+
+/obj/item/storage/belt/military/traitor/hacker/New()
+	..()
+	new /obj/item/screwdriver(src, "red")
+	new /obj/item/wrench(src)
+	new /obj/item/weldingtool/largetank(src)
+	new /obj/item/crowbar/red(src)
+	new /obj/item/wirecutters(src, "red")
+	new /obj/item/multitool/ai_detect(src)
+	new /obj/item/flashlight/emp(src)
```

Comment:
I would put the flashlight outside of the toolbelt otherwise you can miss that it's an EMP. With it in the box it stands out more.